lib/jsonapi/serializer.ex

```diff
@@ -153,11 +153,16 @@ defmodule JSONAPI.Serializer do
 
   @spec encode_relation(tuple()) :: map()
   def encode_relation({rel_view, rel_data, _rel_url, _conn} = info) do
+    encoded_data = encode_rel_data(rel_view, rel_data)
```

Comment:
Here, we capture `encoded_data` so that we can check if it's nil on line 162 in order to suppress rendering of links.

Of course, we will also set the `data:` key of the returned Map.